randovania/gui/dialog/node_details_popup.py

```diff
@@ -303,7 +303,9 @@ def _create_connections_visualizer(
         if parent in self._connections_visualizers:
             self._connections_visualizers.pop(parent).deleteLater()
 
-        self._connections_visualizers[parent] = ConnectionsVisualizer(parent, layout, requirement)
+        self._connections_visualizers[parent] = ConnectionsVisualizer(
```

Comment:
mypy error

Reply:
... these errors give me a feeling something is wrong with mypy